src/components/Settings.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+	<div id="recommendations-setting-enabled">
+		<input id="recommendationsEnabledToggle"
+			v-model="enabled"
+			class="checkbox"
+			checked="checked"
+			type="checkbox"
+			name="enabled">
+		<label for="recommendationsEnabledToggle">{{ t('recommendations', 'Recommendations enabled') }}</label>
```

Comment:
I’d say better would be "Show recommendations" for uniformity. :)
```suggestion
		<label for="recommendationsEnabledToggle">{{ t('recommendations', 'Show recommendations') }}</label>
```

As e.g. if we ever have recommendations on mobile, the settings should be independent and purely dealing with the visibility on that specific client.

Reply:
Besides that it works great! :)

Reply:
Done!